libs/blocks/media/media.css

```diff
@@ -103,13 +103,82 @@ div[class*="-up"] .media .foreground > .media-row {
   padding: 0 0 var(--spacing-m);
 }
 
+.media .default-list {
+  padding-left: 20px;
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+.media li:before {
+  content: "";
+  display: inline-block;
+  margin-left: 3px;
+}
+
+.media.checklist li {
+  background: url("/img/checklist.svg") no-repeat 5px 5px transparent;
+  list-style-type: none;
+  margin-left: -25px;
+  padding: 0 0 1px 29px;
+  vertical-align: middle;
+}
+
+.media .subcopy {
+  font-size: 16px;
+  line-height: 24px;
+  margin-top: 16px;
+  color: #2C2C2C;
+}
+
+.media .action-area {
+  margin-bottom: 0;
+}
+
+.media h3.subcopy-heading {
+  font-size: 16px;
```

Comment:
We should try to use our typography.css classes, rather then setting new font-size values. 
ex. `.heading-xs`

Reply:
resolved @ryanmparrish  https://github.com/adobecom/milo/pull/971/commits/40626583cf40796a247d6a1b33e314ade1fac93c